### PR TITLE
Add terraform steps to deployToTest workflow

### PR DESCRIPTION
**Pull request by mayaspivak**
1. Move terraform files into their own config directory.
2. Refactor into multiple terraform files for readability.
3. Remove scheduler resources from terraform config since they wont be used. The ingestion service can still be triggered manually or through pubsub.
4. Add the test values for the inputs.
5. Add the terraform steps to the workflow.
6. Add condition so that workflow is not run in forked repos. This is currently commented out along with the run on push trigger so that the workflow can be tested.

### CODE REVIEW

File: .github/workflows/deployToTest.yml [hunk 1]
```diff
@@ -39,3 +42,29 @@ jobs:
...
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1
+        working-directory: ./config
+      - name: Terraform Init
+        working-directory: ./config
+        run: terraform init
+      - name: Terraform Format
+        working-directory: ./config
+        run: terraform fmt -check
+      - name: Terraform Plan
```
Comment (jenniebrown) [inline .github/workflows/deployToTest.yml:54]: Is there benefit to running terraform plan here if it's just going to auto approve? Does this give any information to human users?

File: config/run.tf [hunk 2]
```diff
@@ -0,0 +1,107 @@
+/* [BEGIN] Cloud Run Setup */
+
+# Create a Pub/Sub topic to trigger the GCS-to-BQ service.
+resource "google_pubsub_topic" "notify_data_ingested" {
```
Comment (jenniebrown) [inline config/run.tf:4]: I think we also need the upload_to_gcs topic here

File: config/main.tf [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Specify the provider, here
+provider "google" {
+  project     = var.project_id
+  region      = var.compute_region
+  credentials = var.gcp_credentials
```
Comment (jenniebrown) [inline config/main.tf:5]: Are the credentials just so github can deploy via terraform?

**Review by jenniebrown — CHANGES_REQUESTED** [review]

File: config/main.tf (hunk 3 above)
Comment (mayaspivak) [reply config/main.tf:5]: Yep. I downloaded them from the test project and saved them to a github secret.

File: config/run.tf (hunk 2 above)
Comment (mayaspivak) [reply config/run.tf:4]: Good catch, thanks! I must have accidentally deleted it in the refactor.

File: .github/workflows/deployToTest.yml (hunk 1 above)
Comment (mayaspivak) [reply .github/workflows/deployToTest.yml:54]: terraform plan was included in the example starter workflow. I assumed it was necessary to refresh the state before apply, but reading the documentation more closely, it looks like terraform refreshes the state before any command. I will remove it.

**Review by jenniebrown — APPROVED** [review]

**Pull request merged**